client/blocks/image-editor/docs/example.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ImageEditor from '../';
+import { getCurrentUser } from 'state/current-user/selectors';
+
+function ImageEditorExample( { primarySiteId } ) {
+	const media = {
+		URL: 'https://upload.wikimedia.org/wikipedia/commons/thumb/b/b6/SIPI_Jelly_Beans_4.1.07.tiff/lossy-page1-256px-SIPI_Jelly_Beans_4.1.07.tiff.jpg'
```

Comment:
I wanted to add an image which has CORS enabled. Didn't try CloudUp though. Great suggestion!